Dockerfile

```diff
@@ -1,38 +1,26 @@
-# Use an official Python base image from the Docker Hub
 FROM python:3.10-slim
 
-# Install git
-RUN apt-get -y update
-RUN apt-get -y install git chromium-driver
+RUN apt-get update
+RUN apt-get install -y git chromium-driver wget gnupg2 libgtk-3-0 libdbus-glib-1-2 dbus-x11 xvfb ca-certificates
```

Comment:
I'd prefer keeping X11 and the Xvfb etc out of the docker image, as docker containers are headless anyways, and clearly state that only headless browsing is supported in the docker container. Headless browsing should work fine since #1473.